docs.json

```diff
@@ -0,0 +1,88 @@
+{
+    "$schema": "https://mintlify.com/schema.json",
+    "name": "CAMEL Documentation",
+    "theme": "mint",
```

Comment:
Can you help to align the theme with our branding, specifically color from mint to #B5AFFF